examples/cc3220sf/xively_demo/xively_example.c

```diff
@@ -426,6 +426,9 @@ void* xivelyExampleThread( void* arg )
 
         /* shutdown the xively library */
         xi_shutdown();
+
+        sl_Stop(5);
+        simpleLinkMode = sl_Start( NULL, NULL, NULL );
```

Comment:
I am ok with this workaround especially if it works.
Note:
```
\param[in]      Timeout                       Stop timeout in msec. Should be used to give the device time to finish \n 
                                                  any transmission/reception that is not completed when the function was called. \n
```
I just want to double check your intention wasn't 5 sec but msec.

Reply:
I don't think we need to give the device much time to empty its buffers. If we got here, it's because the WiFi dropped anyway. 5ms will do, but thanks for pointing it out :)

Reply:
👍